addons/hr_recruitment_integration_base/models/hr_job_post.py

```diff
@@ -0,0 +1,204 @@
+                ('campaign_end_date', '=', False),
+        ])
+        for post in posts_to_start:
+            wizard = self.env['hr.recruitment.post.job.wizard'].create({
+                'job_id': post.job_id.id,
+                'apply_method': post.apply_method,
+                'platforms': [(6, 0, post.platform.ids)],
+                'campaign_start_date': post.campaign_start_date,
+                'campaign_end_date': post.campaign_end_date,
+                'post': post.post,
+                'post_ids': post.ids,
+                post._contact_point_to_vector(): post.apply_vector,
+            })
+            wizard._post_job()
+
+    def _delete_post(self): ...
+
+    def _contact_point_to_vector(self):
+        self.ensure_one()
+        if self.apply_method == 'email':
+            return 'job_apply_mail'
+        return ''
+
+    def action_update_job_post_check(self):
+        self.ensure_one()
+        if self.platform.price_to_update:
+            raise RedirectWarning(
+                message=_('This action will update the job post on the platform. '
+                            'This action will cost %(price)s credits. Do you want to continue?',
+                            price=self.platform.price_to_update),
+                action=self.env.ref('hr_recruitment_integration_base.hr_job_post_double_check_action').id,
+                additional_context={
+                    'active_id': self.id,
+                    'uid': self.env.user.id,
+                },
+                button_text=_('Update'),
+            )
+        self.action_update_job_post()
+
+    def action_update_job_post(self):
+        self.ensure_one()
+        post_wizard = self.env['hr.recruitment.post.job.wizard'].create({
+            'job_id': self.job_id.id,
+            'apply_method': self.apply_method,
+            'platforms': [(6, 0, self.platform.ids)],
+            'campaign_start_date': self.campaign_start_date,
+            'campaign_end_date': self.campaign_end_date,
+            'post': self.post,
+            'post_ids': self.ids,
+            self._contact_point_to_vector(): self.apply_vector,
+            'api_data': self.api_data,
+        })
+        post_wizard.action_post_job()
+        return {
+            'type': 'ir.actions.client',
+            'tag': 'reload',
+        }
+
+    def action_post_job(self):
+        post_wizard = self.env['hr.recruitment.post.job.wizard'].create({
+            'job_id': self.job_id.id,
+            'apply_method': self.apply_method,
+            'post': self.post,
+        })
+
+        return {
+            'name': _('Reuse Job Post'),
+            'type': 'ir.actions.act_window',
+            'res_model': 'hr.recruitment.post.job.wizard',
+            'res_id': post_wizard.id,
+            'view_mode': 'form',
+            'views': [(False, 'form')],
+            'target': 'new',
+        }
+
+    def action_post_now_multi(self):
```

Comment:
Models method are @api.multi by default.
```suggestion
    def action_post_now(self):
```